draft-ietf-ppm-dap.md

```diff
@@ -2153,20 +2088,13 @@ When aggregation recovers an output share, it must be stored into an appropriate
 "batch bucket", which is defined in this section. The data stored in a batch
 bucket is kept for eventual use in the {{collect-flow}}.
 
-Batch buckets are indexed by a batch mode-specific "batch bucket identifier":
-
-* For the time-interval batch mode, the batch bucket identifier is an interval
-  of time whose beginning is a multiple of the task's `time_precision` and whose
-  duration is equal to the task's `time_precision`. The identifier associated
-  with a given output share is the unique such interval containing the client
-  timestamp of the report the output share was recovered from; for example, if
-  the task's `time_precision` is 1000 seconds and the report's timestamp is
-  1729629081, the relevant batch bucket identifier is `(1729629000, 1000)`. (The
-  first element of the pair denotes the start of the batch interval and the
-  second denotes the duration.)
-* For the leader-selected batch mode, the batch bucket identifier is a batch ID.
-  The identifier associated with a given output share is the batch ID of the
-  related aggregation job.
```

Comment:
Moved to the new sections below; we'll just leave a high level summary below. (Note that this is a point where the batch mode needs to define semantics.)